src/main/java/io/github/mountainrange/mule/gameplay/WorldMap.java

```diff
@@ -193,6 +193,15 @@ public int getRows() {
 		return baseGrid.getRows();
 	}
 
+	/**
+	 * Returns the tile with the town in it.
+	 * @return town The town tile
+	 */
+	public Tile getTown() {
+		//todo
+		return null;
```

Comment:
If you want, we could loop through every tile here, this should probably be removed/implemented though before this is merged